source.yml

```diff
@@ -298,7 +298,7 @@ version_control:
 
     - perception/apriltags:
         type: archive
```

Comment:
I guess we can also make this

    github: AprilRobotics/apriltag

And the manifest can be moved to rock.core/manifests/perception/apriltags.xml, instead of adding it via patch.

Reply:
Great suggestion! Do we still want to keep it pinned to a release tag, or do we want to follow their master branch, though?

Reply:
Agreed.

Reply:
Thanks for your feedback! I added a commit which uses the current github master branch and moves the manifest from the patch into the `manifests`-Folder as Christoph suggested.